### ISSUE TITLE
genomic variation within an indel discrepancy is incorrectly projected as intronic variant

### ISSUE THREAD

**Issue by reece** [biocommons/hgvs#353]
Originally reported by: **Reece Hart (Bitbucket: [reece](http://bitbucket.org/reece), GitHub: [reece](http://github.com/reece))**

----------------------------------------

Related: #259

This issue was discovered when debugging #351.

```
In [15]: vg = hp.parse_hgvs_variant('NC_000012.11:g.122064775C>T')

In [17]: evm.g_to_c(vg, 'NM_032790.3')
Out[17]: SequenceVariant(ac=NM_032790.3, type=c, posedit=126+2C>T)

In [18]: print(hgvs.utils.context.variant_context_w_alignment(evm, vg))
   NM_032790.3  c        108 >                                               >      142 NM_032790.3:c.126+2C>T
   NM_032790.3  n        301 > CGGGGAGCCCCCGGGGGCC------CCGCCACCGCCGCCGT     >      335 NM_032790.3:n.319+2C>T
                               |||||||||||||||||||------||||||||||||||||     19=6D16= 
   NC_000012.11 g  122064755 > CGGGGAGCCCCCGGGGGCCCCGCCACCGCCACCGCCGCCGT     > 122064795 NC_000012.11:g.122064775C>T
   NC_000012.11 g  122064755 < GCCCCTCGGGGGCCCCCGGGGCGGTGGCGGTGGCGGCGGCA     < 122064795 NC_000012.11:g.122064775C>T
                                                   *                          NC_000012.11:g.122064775C>T

```

It's unclear what the result should be here, but an intronic variant is surely incorrect.

----------------------------------------
- Bitbucket: https://bitbucket.org/biocommons/hgvs/issue/353

**Comment by PeteCausey-Freeman** [comment]
Hi Reece,

I have been working on code to identify and tackle issues like this. So far I have had a good degree of success. This is a more complex example that I have worked on previously so I'm glad I spotted it. When you start making changes within the overhaul (#208) please can you keep me updated as the changes could potentially screw up my code. Happy to share information if you are interested.

**Comment by icebert** [comment]
Resolved in hgvs release 1.2.0
